editor/src/messages/portfolio/document/node_graph/node_graph_message_handler/document_node_types.rs

```diff
@@ -1913,6 +1913,51 @@ fn static_nodes() -> Vec<DocumentNodeType> {
 			properties: node_properties::no_properties,
 			..Default::default()
 		},
+		DocumentNodeType {
+			name: "OR",
+			category: "Logic",
+			identifier: NodeImplementation::proto("graphene_core::logic::LogicOrNode<_>"),
+			inputs: vec![
+				DocumentInputType::value("First", TaggedValue::Bool(false), true),
+				DocumentInputType::value("Second", TaggedValue::Bool(false), true),
+			],
+			outputs: vec![DocumentOutputType::new("Output", FrontendGraphDataType::Boolean)],
+			properties: node_properties::logic_operator_properties,
+			..Default::default()
+		},
+		DocumentNodeType {
+			name: "AND",
```

Comment:
Thoughts on capitalisation @Keavon? They are often capitalised in logic but look slightly out of place with the other title case nodes.

Reply:
I think Title Case is best for all of these except XOR since the other three are English words.